be/src/storage/olap_common.h

```diff
@@ -305,6 +305,8 @@ struct RowsetId {
         }
     }
 
+    int64_t id() { return hi & LOW_56_BITS; }
```

Comment:
Why not set the state of tablet? The dropped tablet should wait the rowset been de-refereneced.

Reply:
> Why not set the state of tablet? The dropped tablet should wait the rowset been de-refereneced.

The state of tablet is already set to `SHUTDOWN` before remove the rowset.